test/Microsoft.AspNetCore.Mvc.TestCommon/SimpleValueProviderFactory.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Internal;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding
+{
+    public class SimpleValueProviderFactory : IValueProviderFactory
```

Comment:
Put this and `SimpleValueProvider` in the test common project because they are used multiple places